drivers/src/memory_layout.rs

```diff
@@ -25,25 +25,26 @@ pub const ROM_ORG: u32 = 0x00000000;
 pub const MBOX_ORG: u32 = 0x30000000;
 pub const ICCM_ORG: u32 = 0x40000000;
 pub const DCCM_ORG: u32 = 0x50000000;
-pub const MAN1_ORG: u32 = 0x50000000;
-pub const MAN2_ORG: u32 = 0x50001800;
-pub const FHT_ORG: u32 = 0x50003000;
-pub const LDEVID_TBS_ORG: u32 = 0x50003800;
-pub const FMCALIAS_TBS_ORG: u32 = 0x50003C00;
-pub const RTALIAS_TBS_ORG: u32 = 0x50004000;
-pub const PCR_LOG_ORG: u32 = 0x50004400;
-pub const FUSE_LOG_ORG: u32 = 0x50004800;
-pub const BOOT_STATUS_ORG: u32 = 0x50004BE4;
-pub const CFI_VAL_ORG: u32 = 0x50004BE8;
-pub const CFI_MASK_ORG: u32 = 0x50004BEC;
-pub const CFI_XO_S0_ORG: u32 = 0x50004BF0;
-pub const CFI_XO_S1_ORG: u32 = 0x50004BF4;
-pub const CFI_XO_S2_ORG: u32 = 0x50004BF8;
-pub const CFI_XO_S3_ORG: u32 = 0x50004BFC;
-pub const DATA_ORG: u32 = 0x50004C00;
-pub const STACK_ORG: u32 = 0x5001A000;
-pub const ESTACK_ORG: u32 = 0x5001F800;
-pub const NSTACK_ORG: u32 = 0x5001FC00;
+// Region of ~ 1k bytes between DCCM_ORG and CFI_VAL_ORG is reserved for ROM's DATA_ORG
+pub const CFI_VAL_ORG: u32 = 0x500003E4;
+pub const CFI_MASK_ORG: u32 = 0x500003E8;
+pub const CFI_XO_S0_ORG: u32 = 0x500003EC;
+pub const CFI_XO_S1_ORG: u32 = 0x500003F0;
+pub const CFI_XO_S2_ORG: u32 = 0x500003F4;
+pub const CFI_XO_S3_ORG: u32 = 0x500003F8;
+pub const BOOT_STATUS_ORG: u32 = 0x500003FC;
+pub const MAN1_ORG: u32 = 0x50000400;
+pub const MAN2_ORG: u32 = 0x50001C00;
+pub const FHT_ORG: u32 = 0x50003400;
+pub const LDEVID_TBS_ORG: u32 = 0x50003C00;
+pub const FMCALIAS_TBS_ORG: u32 = 0x50004000;
+pub const RTALIAS_TBS_ORG: u32 = 0x50004400;
+pub const PCR_LOG_ORG: u32 = 0x50004800;
+pub const FUSE_LOG_ORG: u32 = 0x50004C00;
+pub const DATA_ORG: u32 = 0x50005000;
+pub const STACK_ORG: u32 = 0x5001A400;
+pub const ESTACK_ORG: u32 = 0x5001FC00;
+pub const NSTACK_ORG: u32 = 0x50020000;
```

Comment:
0x50020000 is the end of DCCM; you can't move this.

Reply:
Moved STACK_ORG, ESTACK_ORG, and NSTACK_ORG back to original addresses, and reduced size of DATA_ORG by 1k bytes to accommodate the new reserved section at the start of DCCM.